frontend/src/metabase/query_builder/components/view/sidebars/SummarizeSidebar/SummarizeSidebar.tsx

```diff
@@ -55,20 +46,16 @@ export function SummarizeSidebar({
       <SummarizeAggregationItemList
         px="lg"
         query={query}
+        onQueryChange={onAggregationChange}
         stageIndex={stageIndex}
-        aggregations={aggregations}
-        onQueryChange={handleQueryChange}
       />
       <Divider my="lg" />
       {hasAggregations && (
         <SummarizeBreakoutColumnList
           px="lg"
           query={query}
+          onQueryChange={onUpdateQuery}
```

Comment:
It's worth making the naming consistent

```suggestion
          onQueryChange={onQueryChange}
```